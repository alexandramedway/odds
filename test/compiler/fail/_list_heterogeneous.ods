do ["one", "of", "these", "things", true, "is", "not", "like", "the", "others"]

