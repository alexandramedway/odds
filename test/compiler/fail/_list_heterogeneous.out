Analyzer Exception: Invalid element of type Bool in list of type List[String]
